mobile/components/CommentModal.tsx

```
@@ -0,0 +1,135 @@
+                                        className='w-full h-48 rounded-2xl mb-3'
+                                        resizeMode='cover'
+                                    />
+                                )}
+                            </View>
+                        </View>
+                    </View>
+
+                    {/*  Comments Section */}
+
+                    {selectedPost.comments.map((comment) => (
+                        <View className='flex-row'>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)